deb/changes.go

```diff
@@ -186,6 +186,18 @@ func (c *Changes) PackageQuery() (PackageQuery, error) {
 			}
 		}
 
+		// matching debug ddeb packages, they're not present in the Binary field
+		var ddebQuery PackageQuery
+		ddebQuery = &AndQuery{
+			L: &FieldQuery{Field: "Name", Relation: VersionPatternMatch, Value: "*-dbgsym"},
```

Comment:
I think that's too broad here, I would suggest to use `for` loop above to add `binary + "-dbgsym"` query

Also unit-tests are failing.

Thanks for the PR!